modules/custom/openy_programs_search/src/Plugin/Block/ProgramsSearchBlock.php

```diff
@@ -110,9 +122,17 @@ public function blockForm($form, FormStateInterface $form_state) {
       '#title' => $this->t('Categories Config'),
     ];
 
+    try {
+      $categories = $this->storage->getCategories();
+    }
+    catch (\Exception $e) {
+      $categories = [];
+      $this->logger->error($e->getMessage());
```

Comment:
Could you also add a message to the user about why the form is empty and where to look for proper settings?